WooCommerce/Classes/ViewRelated/Dashboard/Reviews/ReviewsDashboardCardViewModel.swift

```diff
@@ -48,6 +54,11 @@ final class ReviewsDashboardCardViewModel: ObservableObject {
                                                                            matching: nil,
                                                                            fetchLimit: Constants.numberOfItems,
                                                                            sortedBy: [])
+
+        self.notificationsResultsController = ResultsController<StorageNote>(storageManager: storageManager,
+                                                                           sectionNameKeyPath: "normalizedAgeAsString",
```

Comment:
Is it necessary?

Reply:
This entire ResultsController was copied exactly from `ReviewsDataSource`

https://github.com/woocommerce/woocommerce-ios/blob/0235248c87238e55c674d014c425a05870ada14e/WooCommerce/Classes/ViewRelated/Reviews/ReviewsDataSource.swift#L48

The particular code was initially added almost six years ago in this commit https://github.com/woocommerce/woocommerce-ios/commit/01b0281c0581d8b6c9a889979ecc8037ec15b9ac

I did not investigate if it's deprecated along the way and needs to be removed. If it is, `ReviewsDataSource` will need to be updated too (perhaps we can make a separate issue about it).

Reply:
I think it's necessary to understand what `sectionNameKeyPath` is for. You can look it up for more detailed explanations, but it's essentially used to group the models into sections by a specified key path. This is necessary for the reviews list because we're displaying the list in sections based on the age of the reviews. In our case, we display the reviews list without grouping them, so using `sectionNameKeyPath` is redundant.

Reply:
Yup! I did eventually investigate yesterday and removing it seems to remove the issue (mentioned this in our Slack discussion).